src/pages/about-us/_OurLeadership.tsx

```diff
@@ -0,0 +1,351 @@
+            ...fadeIn
+        }
+        derek: file(relativePath: { eq: "about-us/derek.jpg" }) {
+            ...fadeIn
+        }
+        gary: file(relativePath: { eq: "about-us/gary.jpg" }) {
+            ...fadeIn
+        }
+        jennice: file(relativePath: { eq: "about-us/jennice.jpg" }) {
+            ...fadeIn
+        }
+        jeya: file(relativePath: { eq: "about-us/jeya.jpg" }) {
+            ...fadeIn
+        }
+        jy: file(relativePath: { eq: "about-us/jy.jpg" }) {
+            ...fadeIn
+        }
+        joanna: file(relativePath: { eq: "about-us/joanna.jpg" }) {
+            ...fadeIn
+        }
+        louise: file(relativePath: { eq: "about-us/louise.jpg" }) {
+            ...fadeIn
+        }
+        rakshit: file(relativePath: { eq: "about-us/rakshit.jpg" }) {
+            ...fadeIn
+        }
+        raunak: file(relativePath: { eq: "about-us/raunak.jpg" }) {
+            ...fadeIn
+        }
+        seema: file(relativePath: { eq: "about-us/seema.jpg" }) {
+            ...fadeIn
+        }
+        shyamala: file(relativePath: { eq: "about-us/shyamala.jpg" }) {
+            ...fadeIn
+        }
+        tom: file(relativePath: { eq: "about-us/tom.jpg" }) {
+            ...fadeIn
+        }
+        waqas: file(relativePath: { eq: "about-us/waqas.jpg" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const StyledSectionContainer = styled(SectionContainer)`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0;
+    @media ${device.tablet} {
+        padding: 0 16px 40px;
+    }
+`
+
+const StyledCssGrid = styled(CssGrid)`
+    max-width: 996px;
+
+    @media (max-width: 1100px) {
+        max-width: 90%;
+        padding: 0 40px;
+    }
+    @media ${device.tablet} {
+        max-width: unset;
+        padding: 0;
+        grid-template-columns: ;
+    }
+`
+
+const StyledHeader = styled(Header)`
+    padding-bottom: 40px;
+    @media ${device.laptop} {
+        font-size: 28px;
+    }
+    @media ${device.tablet} {
+        padding-bottom: 24px;
+    }
+`
+const ModalFlex = styled(Flex)`
+    position: absolute;
+    top: 130px;
+    background-color: white;
+    z-index: 1;
+    border-radius: 5px;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 4px 15px;
+    white-space: pre;
+    width: fit-content;
+    @media ${device.tablet} {
+        top: 124px;
+    }
+    @media ${device.tabletS} {
+        top: 108px;
+    }
+    @media ${device.mobileL} {
+        top: 98px;
+    }
+`
+type StyledImageWrapperPropsType = ImageWrapperProps & {
+    onMouseOver: (event: FocusEvent<any>) => void
+    onMouseLeave: (event: FocusEvent<any>) => void
+    tabindex: string
+}
+const StyledImageWrapper = styled(ImageWrapper)<StyledImageWrapperPropsType>`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    padding-bottom: 20px;
+    @media ${device.tabletS} {
+        width: 104px;
+        height: 124px;
+    }
```

Comment:
Why are we extending `ImageWrapper` here when `QueryImage` itself has already an imagewrapper inside?

Reply:
we need one more container with height> QueryImage to move mouse to the popup div